app/src/main/java/com/asfoundation/wallet/ui/iab/MergedAppcoinsPresenter.kt

```diff
@@ -146,20 +164,10 @@ class MergedAppcoinsPresenter(private val view: MergedAppcoinsView,
     }
   }
 
-  private fun getCreditsBalance(): Observable<FiatValue> {
-    return balanceInteract.getCreditsBalance()
-        .map { it.second }
-  }
-
-  private fun getAppcBalance(): Observable<FiatValue> {
-    return balanceInteract.getAppcBalance()
-        .map { it.second }
+  private fun getCreditsBalance(): Observable<FiatValue> =
+      mergedAppcoinsInteract.getCreditsBalance()
 
-  }
+  private fun getAppcBalance(): Observable<FiatValue> = mergedAppcoinsInteract.getAppcBalance()
 
-  private fun getEthBalance(): Observable<FiatValue> {
-    return balanceInteract.getEthBalance()
-        .map { it.second }
-
-  }
+  private fun getEthBalance(): Observable<FiatValue> = mergedAppcoinsInteract.getEthBalance()
```

Comment:
Explicit type can be removed

Reply:
Isn't it better to leave the type to better readability and accessibility to the type?

Reply:
we can find the type with autocomplete hints and by moving the mouse over. but it's up to you